examples/hetzner-deploy/make_oc10_apps.sh

```diff
@@ -206,7 +206,7 @@ for param in \$PARAM; do
 	occ config:system:set trusted_domains 2 --value="\$wopi_fqdn"
 	echo >> ~/POSTINIT.msg "RICHDOCUMENTS: The following manual steps are needed to use richdocuments"
 	echo >> ~/POSTINIT.msg "RICHDOCUMENTS:  - Register at dash.cloudflare.com:  $IPADDR \$wopi_fqdn"
-	echo >> ~/POSTINIT.msg "RICHDOCUMENTS:  - To get a certificate, run:        certbot -m qa@owncloud.com --no-eff-email --agree-tos -d \$wopi_fqdn"
+	echo >> ~/POSTINIT.msg "RICHDOCUMENTS:  - To get a certificate, run:        certbot -m qa@example.com --no-eff-email --agree-tos -d \$wopi_fqdn"
```

Comment:
I've confirmed I can generate a cert with that emai.
The change implies, that letsencrypt cannot contact us. The man page says
`-m EMAIL         Email address for important account notifications`

Would it violate any rules?

Reply:
well you should 1st not use productive Let's Encrypt servers for testing see:
-  https://letsencrypt.org/docs/staging-environment/
Please use their staging environment

Second - you should have a private copy of this repo containing your private information - like mail addresses strings etc.